03-getting-started/03-configuration-and-env.adoc

```diff
@@ -86,6 +86,26 @@ Env.get('APP_SECRET')
 NOTE: `Env.get` always returns a string. If you want to get a boolean value you'll need to check it as follows. +
   `Env.get('...') === 'true'`
 
+It is highly recommended to contain access to environment variables to configuration files and not to use `Env` provider anywhere else in the app.
+
+==== Failing fast if environment variable does not exist
+
+When you have an environment variable that is essential for running the application, you may use a `getOrFail` method which will throw an Error at application boot time, if the variable is not set.
```

Comment:
Saying `will throw an Error at application boot time` is not true, since some people have been using `Env` deep inside their code.

So we should make clear that in order to avoid missing Env variables, it is recommended to use them inside `config` and then exceptions will be raised at boot time